src/components/common/Field.tsx

```diff
@@ -131,3 +132,48 @@ export const SelectField: React.FC<SelectFieldProps> = ({
     </div>
   );
 };
+
+type CheckboxFieldProps = {
+  fieldClassName?: string;
+  tip?: string;
+  error?: string;
+} & CheckboxProps &
+  HTMLProps<HTMLSelectElement>;
```

Comment:
HTMLSelectElement? Shouldn't this be HTMLInputElement (since a checkbox is an `<input>`)? Hopefully this should also solve the typing mismatch for `value`

Reply:
Thanks for catching this, fixed

Reply:
unfortunately, the states for checkbox are still `[]` and `["on"]` ☹️